### DISCUSSION TITLE
How to hide or change url

### DISCUSSION THREAD

**Discussion by lucaslomeu** [vercel/next.js#39047] [Help]
I configured my project so that the scroll to the component occurs, but my URL is looking like this: 

`http://localhost:3000/#projects`

How can I remove this # or just leave website.com

**Comment by GabenGar** [comment]
Don't use hashes for scrolling then.

**Reply by lucaslomeu** [reply]
I'm using all components rendered in app.js and using Link's href to navigate to the component.

**Reply by GabenGar** [reply]
For what purpose?

**Reply by lucaslomeu** [reply]
App

```
export default function Home () {
  return (
    <div>
      <Navbar />
        <div>
          <About />
          <Experience />
          <Projects />
          <Blog />
          <Footer />
      </div>
    </div>
  )
}
```


NavBar Link:
```
 <Link href='#projetos' smooth={true} duration={500} offset={50}>
            <a className='link-menu'>Projetos</a>
 </Link>
```

If I take out the #, it doesn't go to the element.

**Reply by GabenGar** [reply]
Why not just switch them with buttons?

**Reply by lucaslomeu** [reply]
But how would I make it scroll to the element and add /route to the url?

**Reply by GabenGar** [reply]
Just make them separate pages, it is nextjs after all. As for scrolling there are [`Window.scrollTo()`](https://developer.mozilla.org/en-US/docs/Web/API/Window/scrollTo) and [`HTMLElement.focus()`](https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/focus).

**Reply by lucaslomeu** [reply]
When you say make pages separate, do you mean removing all app.js components? I still don't understand how I would make it so when I click on a link the page will slide.

The website is: www.lucaslomeu.com

**Reply by GabenGar** [reply]
Yes, I mean making those sections as separate pages. Don't need sliding in that case.
